### ISSUE TITLE
[Feature Request] Texture Filters; Button Functions; Load/Save States and MORE

### ISSUE THREAD

**Issue by dnyce13** [weihuoya/citra#490]
Hello, I want to start off saying keep up the amazing work. This emulator is awesome! My well thought out requests:

Adding Texture Filters as an option

Add more utility options in the Quick menu such as:
-Swap between screens
-Reset Game
-Load/Save States
-Autosave state option(whenever user backs out of emulator)

Add special virtual buttons and external control map bindings
-Fast forward (3 different speed selections based on the tap of the button)
-Rewind (1 speed setting)
-Simulate microphone noise
-Quick load/Save state button
-Quick swap between screens
-Cycle thru screen layouts

Ability to scale virtual button mappings individually

Ability when using external controller to map the right stick as a "mouse" and clicking the stick for interaction.

Currently in pokemon ultra sun/ultra moon one cannot interact with the pokemon when petting or taking care of it. Have not tested this in the other pokemon games to confirm if it's game specific OR emulator specific.

That's all! Keep up the good work 👏.

**Comment by eadmaster** [comment]
Just found this other fork with savestates and cheat support:
https://github.com/Gamer64ytb/Citra-Enhanced/

**Comment by dnyce13** [comment]
Yes, but they're not updating it as often as Citra MMJ is. So I'm caught between wanting better game play with MMJ and more optimizations or QOL with Enhanced with load and save states.On Aug 6, 2022 7:28 AM, eadmaster ***@***.***> wrote:
Just found this other fork with savestates and cheat support:
https://github.com/Gamer64ytb/Citra-Enhanced/
